var/spack/repos/builtin/packages/py-uproot/package.py

```diff
@@ -16,11 +16,36 @@ class PyUproot(PythonPackage):
     on C++ ROOT. Instead, it uses Numpy to cast blocks of data from the ROOT
     file as Numpy arrays."""
 
-    homepage = "https://github.com/scikit-hep/uproot4"
-    pypi     = "uproot/uproot-4.0.6.tar.gz"
+    homepage = "https://uproot4.readthedocs.io"
+    git      = "https://github.com/scikit-hep/uproot4"
+    url      = "https://github.com/scikit-hep/uproot4/archive/0.0.27.tar.gz"
```

Comment:
Why switch to GitHub? PyPI is preferred for Python libraries. Also, the checksums may differ.

Reply:
That is what was in the py-uproot4 package.

Reply:
Personally I would switch both to PyPI, although I don't care too much about converting deprecated, soon to be removed packages.

Reply:
> Also, the checksums may differ.

Good point. That should only affect the newer version that I kept from the original py-uproot.

Reply:
Looking at the py-awkward an py-awkward1 packages, py-awkward1 uses git and py-awkward uses pypi.

@vvolkl Do we want to standardize on git or pypi for awkward and uproot?

Reply:
yes, I think we just didn't bother for awkward1 as it was deprecated, but it makes sense to use pypi

Reply:
The "official" Spack policy is to standardize on PyPI whenever possible: https://spack.readthedocs.io/en/latest/build_systems/pythonpackage.html#pypi-vs-github

If PyPI only has wheels, then I use GitHub instead.